### PR TITLE
Tracking tags

### PR DESCRIPTION
**Pull request by podviaznikov**
Each instance of `Swarm` or `Docker` can be initialized with extra tags that would be passed to https://github.com/Runnable/dogerode/blob/master/index.js on each call

### CODE REVIEW

File: lib/base.js [hunk 1]
```diff
@@ -83,7 +85,8 @@ class BaseDockerClient {
       service: opts.serviceName,
       targetType: opts.targetType,
       host: process.env.DATADOG_HOST,
-      port: process.env.DATADOG_PORT
+      port: process.env.DATADOG_PORT,
+      datadogTags: opts.datadogTags || {}
```
Comment (anandkumarpatel) [inline lib/base.js:89]: +1 but you dont have a test for datadogTags ?

**Review by anandkumarpatel — CHANGES_REQUESTED** [review]

File: lib/base.js (hunk 1 above)
Comment (podviaznikov) [reply lib/base.js:89]: will add it. Good point

Comment (podviaznikov) [reply lib/base.js:89]: done

**Comment by anandkumarpatel** [comment]
Approved

**Review by anandkumarpatel — APPROVED** [review]

**Pull request merged**